scripts/events_module/relationship/pregnancy_events.py

```diff
@@ -880,21 +880,21 @@ def get_kits(
                     ]
                     y = random.randrange(0, 15)
                     start_relation = Relationship(the_cat, kit, False, True)
-                    start_relation.like += parent_to_kit[RelValue.LIKE] + y
-                    start_relation.comfort = parent_to_kit[RelValue.COMFORT] + y
-                    start_relation.respect = parent_to_kit[RelValue.RESPECT] + y
-                    start_relation.trust = parent_to_kit[RelValue.TRUST] + y
+                    start_relation.like = parent_to_kit[RelType.LIKE] + y
```

Comment:
this was += but now isn't? am i having a grandpa moment where i asked for a change and am now un-asking for it